packages/web-components/fast-foundation/src/design-token/interfaces.ts

```diff
@@ -3,7 +3,7 @@
  * or arbitrary observable properties.
  * @public
  */
-export type DerivedDesignTokenValue<T> = T extends Function
+export type DerivedDesignTokenValue<T> = [T] extends [Function]
```

Comment:
Awesome, I didn't know about this feature! Good catch.

Reply:
Would it be a good idea to cherry pick the tests into main to prevent any regressions?